code/__DEFINES/achievements.dm

```diff
@@ -42,7 +42,6 @@
 #define BOSS_MEDAL_LEGION	  "Legion Killer"
 #define BOSS_MEDAL_TENDRIL	  "Tendril Exterminator"
 #define BOSS_MEDAL_SWARMERS   "Swarmer Beacon Killer"
-#define BOSS_MEDAL_KINGGOAT   "King Goat Killer"
```

Comment:
Leave the achievements in for now, i'll have to add some code to make removing them from metadata more graceful